pytest_reportportal/service.py

```diff
@@ -284,23 +284,24 @@ def start_pytest_item(self, test_item=None):
         if self.rp is None:
             return
 
+        parent_item_id = None
```

Comment:
```suggestion
        parent_item_id = self.parent_item_id
```
With the above suggestion, you can remove clauses to determine what value to use in the API calls for `parent_item_id`. It would be safe to just use `parent_item_id` in those cases (line 298 & 316)